metplotpy/contributed/weather_regime/plot_weather_regime.py

```diff
@@ -0,0 +1,99 @@
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+from pylab import *
+import cmocean
+from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from cartopy.util import add_cyclic_point
+import numpy as np
+import sys
+sys.path.insert(0, "/glade/u/home/kalb/UIUC/METcalcpy/")
+import metcalcpy.util.utils as util
+
+def plot_elbow(K,d,mi,line,curve,plot_title,output_plotname):
+
+    plt.plot(K[int(mi)],d[mi]*-1,'*k')
+    plt.plot(K, np.array(curve), 'kx-')
+    plt.plot(K, line, 'r')
+    plt.plot(K, d*-1, 'b')
+    plt.xlabel('k')
+    plt.ylabel('Sum_of_squared_distances')
+    plt.title(plot_title)
+
+    fmt = 'png'
+    full_output_plot = output_plotname + "." + fmt
+    plt.savefig(full_output_plot,format=fmt,dpi=400, bbox_inches='tight')
+
+
+def plot_eof(eof,wrnum,variance_fractions,lons,lats,output_plotname,plevels):
+
```

Comment:
missing python docs with description of input variables.

Reply:
The docs are in a README file now in github.  Are there other docs needed?